[2025-01-12T13:00:15.895+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:15.901+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:00:15.914+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:15.912+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:15.982+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:15.965+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:00:15.985+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:16.106+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.234 seconds
[2025-01-12T13:00:46.554+0000] {processor.py:186} INFO - Started process (PID=137) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:46.555+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:00:46.558+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:46.558+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:46.578+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:00:46.574+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:00:46.580+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:00:46.606+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.059 seconds
[2025-01-12T13:01:17.313+0000] {processor.py:186} INFO - Started process (PID=202) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:17.315+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:01:17.317+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:17.317+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:17.340+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:17.336+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:01:17.345+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:17.370+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.065 seconds
[2025-01-12T13:01:47.431+0000] {processor.py:186} INFO - Started process (PID=267) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:47.432+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:01:47.434+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:47.434+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:47.446+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:01:47.444+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:01:47.449+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:01:47.465+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.039 seconds
[2025-01-12T13:02:17.855+0000] {processor.py:186} INFO - Started process (PID=332) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:17.857+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:02:17.859+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:17.859+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:17.873+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:17.870+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:02:17.874+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:17.902+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.052 seconds
[2025-01-12T13:02:48.353+0000] {processor.py:186} INFO - Started process (PID=397) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:48.362+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:02:48.364+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:48.364+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:48.376+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:02:48.374+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:02:48.378+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:02:48.393+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.044 seconds
[2025-01-12T13:03:18.439+0000] {processor.py:186} INFO - Started process (PID=462) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:18.440+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:03:18.442+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:18.442+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:18.456+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:18.453+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:03:18.458+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:18.485+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.050 seconds
[2025-01-12T13:03:48.670+0000] {processor.py:186} INFO - Started process (PID=527) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:48.672+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:03:48.675+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:48.675+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:48.695+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:03:48.692+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:03:48.697+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:03:48.726+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.062 seconds
[2025-01-12T13:04:19.186+0000] {processor.py:186} INFO - Started process (PID=592) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:19.187+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:04:19.189+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:19.189+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:19.202+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:19.199+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:04:19.203+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:19.228+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.047 seconds
[2025-01-12T13:04:35.345+0000] {processor.py:186} INFO - Started process (PID=651) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:35.346+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:04:35.347+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:35.347+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:35.379+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:04:35.376+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:04:35.381+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:04:35.400+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.060 seconds
[2025-01-12T13:05:05.764+0000] {processor.py:186} INFO - Started process (PID=716) to work on /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:05:05.765+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/__pycache__/Capstone_project.py for tasks to queue
[2025-01-12T13:05:05.767+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:05:05.767+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:05:05.783+0000] {logging_mixin.py:190} INFO - [2025-01-12T13:05:05.779+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/__pycache__/Capstone_project.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/__pycache__/Capstone_project.py", line 4, in <module>
    from capstone_project.etl import extract, transform, load  # Import your ETL functions
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'capstone_project'
[2025-01-12T13:05:05.784+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/__pycache__/Capstone_project.py
[2025-01-12T13:05:05.800+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/__pycache__/Capstone_project.py took 0.041 seconds
